tardis/resources/drone.py

```diff
@@ -68,6 +68,10 @@ def demand(self) -> float:
     def demand(self, value: float):
         self._demand = value
 
+    @property
+    def drone_heartbeat_interval(self) -> int:
```

Comment:
As we are in the scope of a drone already, shouldn't this better be called `heartbeat_interval`?

Reply:
Agreed, I will change https://github.com/MatterMiners/tardis/blob/e4e35068c1c083a40ef0f0a878bd9f5c93e6545f/tardis/resources/drone.py#L76 as well then.

Reply:
Done!